core/proto/keytransparency_v1_grpc/keytransparency_v1_admin_grpc.proto

```diff
@@ -87,3 +73,32 @@ service KeyTransparencyAdminService {
     };
   }
 }
+
+// The AccountRecoveryService API holds a set of account recovery keys.  
+// Access to these APIS are logged and restricted to properly authenticated
+// and authorized callers.  Only accounts that have put one of the recovery
+// keys in their set of authorized_keys can be administered.  Third parties can
+// implement this API in order to provide trust agility with regard to account
+// reset.
+// 
+// Key Rotation: 
+// The set of account recovery keys may rotate over time with new keys being
+// added every X months, X * N months before they become the currently active signing
+// key, and another Y months before they are deactivated from the list of valid
+// verifying keys. In order to maintain the ability of this service to
+// administer accounts, they must refresh their set of authorized public keys,
+// ideally every X months.
+service AccountRecoveryService {
+
+  // GetAuthorizedKeys returns the public keys associated with this service, and the currently active key.
+  rpc GetAutorizedKeys (keytransparency.v1.proto.GetAuthorizedKeysRequest) returns (keytransparency.v1.proto.GetAuthorizedKeysResponse) {}
+
+  // CreateAccount creates a new account and initializes it with data and authorized_keys.
+  // If the account already exists, this operation will fail.
+  rpc CreateAccount(keytransparency.v1.proto.CreateAccountRequest) returns (keytransparency.v1.proto.CreateAccountResponse) {}
+  // SetAccountData sets the public key for an account. 
+  rpc SetAccountData(keytransparency.v1.proto.SetAccountDataRequest) returns (keytransparency.v1.proto.SetAccountDataResponse) {}
```

Comment:
Maybe call it SetAccountPublicKey, if it's a specialised method just for that.  An alternative would be just a standard, masked UpdateAccount method.  Depends on the intent.